README.md

```diff
@@ -11,6 +10,12 @@ The HTTP listener redirects to HTTPS.
 
 The HTTPS listener uses a certificate stored in ACM or IAM.
 
+## Terraform Versions
+
+Terraform 0.12. Pin module version to ~> 3.0. Submit pull-requests to master branch.
```

Comment:
We probably shouldn't suggest people auto-upgrade on minor version, but instead suggest it happens on patch. They'll likely get bitten if they follow this precisely.

```suggestion
Terraform 0.12. Pin module version to ~> 3.0.0. Submit pull-requests to master branch.
```